site/sprints/index.md

```diff
@@ -3,8 +3,6 @@ layout: page
 title: Sprints
```

Comment:
Same comment for the .md file buth with html in it.

Reply:
Resolved